common/changes/@bentley/presentation-common/presentation-typeName-lowercase_2021-03-02-13-38.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/presentation-common",
+      "comment": "Added a property `extendedType` in `PropertyType` interface.",
```

Comment:
```suggestion
      "comment": "Added an [PropertyInfo.extendedType]($presentation-common) attribute.",
```

Reply:
Is this really necessary? These links won't work outside our documentation, for example, [here](https://github.com/imodeljs/imodeljs/blob/master/presentation/common/CHANGELOG.md).

Reply:
Is this really necessary? These links won't work outside our documentation, for example, [here](https://github.com/imodeljs/imodeljs/blob/master/presentation/common/CHANGELOG.md).

Reply:
Good point, thanks. Updated the suggestions.